libs/hmpb/src/generate_fixtures.ts

```diff
@@ -123,6 +127,28 @@ async function generateNhGeneralElectionFixtures(renderer: Renderer) {
   }
 }
 
+async function generateTimingMarkGridOnlyFixtures(
+  renderer: Renderer,
+  paperSize: HmpbBallotPaperSize,
+  fixtureDir: string
+) {
+  const outputDir = normalize(fixtureDir);
+  const specPaths = timingMarkGridOnlyFixtures.specPaths({ paperSize });
+  const specDir = join(outputDir, specPaths.dir);
+  assert(specDir.startsWith(`${outputDir}/`));
+  await rm(specDir, {
+    recursive: true,
+    force: true,
+  });
+  const generated = await timingMarkGridOnlyFixtures.generate(renderer, {
+    paperSize,
+  });
+  const pdfPath = join(outputDir, specPaths.pdf);
+  assert(pdfPath.startsWith(`${outputDir}/`));
```

Comment:
I noticed you've started using `node:assert` instead of the one from `@votingworks/basics`. Any reason why?